packages/analytics-browser/generated/amplitude-snippet.js

```diff
@@ -89,4 +89,4 @@
     };
     window.amplitude = amplitude;
   }
-}(window, document);
\ No newline at end of file
+}(window, document);
```

Comment:
ok to keep but want to note that diffs on this file will be overwritten by the publish action